src/python/WMCore/Services/DBS/DBS3Reader.py

```diff
@@ -592,8 +597,8 @@ def listFileBlockLocation(self, fileBlockName, dbsOnly = False, phedexNodes=Fals
                 msg += "%s\n" % formatEx3(ex)
                 raise DBSReaderError(msg)
 
-            if not any(blockInfo.values()): # no data location from dbs
-                return list()
+        if not any(blockInfo.values()): # no data location
```

Comment:
any(blockInfo) would be enough here Andres, we can get rid of the values method.
Otherwise, changes to this method sounds good to me.